vNext/src/OrleansTestingHost/OrleansTestingHost.csproj

```diff
@@ -55,15 +55,9 @@
     <Compile Include="..\..\..\src\OrleansTestingHost\Extensions\TestConfigurationExtensions.cs">
       <Link>Extensions\TestConfigurationExtensions.cs</Link>
     </Compile>
-    <Compile Include="..\..\..\src\OrleansTestingHost\OrleansTestSecrets.cs">
```

Comment:
Since these are breaking the vNext solution, I wonder if we should cherry pick this commit into a separate PR, unless this PR is ready to merge (all tests passing, etc)